renku/models/datasets.py

```diff
@@ -138,16 +122,22 @@ def _now(self):
         return datetime.datetime.utcnow()
 
 
-_deserialize_files = partial(_deserialize_dict, cls=DatasetFile)
-
-
 def _parse_date(value):
     """Convert date to datetime."""
     if isinstance(value, datetime.datetime):
         return value
     return parse_date(value)
 
 
+def _convert_dataset_files(value):
```

Comment:
Should we add absolute path here?

Reply:
no

Reply:
Why?

Reply:
there is no need for an absolute path anywhere. we keep all paths relative to `client` or metadata file

Reply:
```there is no need for an absolute path anywhere``` 

Not really true, now is it? ;)